src/CNGLOB.H

```diff
@@ -99,6 +100,7 @@ typedef PVOID HANDLE;
 typedef HANDLE HWND;
 typedef HANDLE HINSTANCE;
 typedef HINSTANCE HMODULE;
+#define CSE_MAX_PATH 2048
```

Comment:
I would put the two dependent definitions of CSE_MAX_PATH together for clarity, and then separately conditionally define all the other typedefs.